01-ActivityModel/activity-model/config/epc_api.yaml

```diff
@@ -0,0 +1,25 @@
+################################################################################
+# Insert in this file the necessary information to download EPC data using API.
+#
+################################################################################
+
+#  EPC credentials
+epc_user: "user@email"
+epc_key: "user_key"
```

Comment:
In the readme I have included additional description, where users can understand epc_key means, the `key_api` https://epc.opendatacommunities.org/docs/api